sormas-ui/src/main/java/de/symeda/sormas/ui/utils/NumberValidator.java

```diff
@@ -28,9 +28,11 @@
 
 	private BigDecimal minValue;
 	private BigDecimal maxValue;
+	private boolean decimalAllowed;
 
 	public NumberValidator(String errorMessage) {
 		super(errorMessage);
+		decimalAllowed = true;
 	}
 
 	public NumberValidator(String errorMessage, Number minValue, Number maxValue) {
```

Comment:
You can remove the code from this constructor and instead call the new one like this to avoid redundancies:

`this(errorMessage, minValue, maxValue, true);`